autobuild-ng.sh

```diff
@@ -2,8 +2,10 @@
 
 if [ -z "$1" ]; then
 	export ARCH=x86_64
```

Comment:
i sholud probably change this to not be x86 but uname -m but later

Reply:
Went ahead and added it